.github/workflows/deno.yml

```diff
@@ -28,3 +28,6 @@ jobs:
 
       - name: Run tests
```

Comment:
What about changing this step to invoke `deno task test` as defined in your `deno.jsonc`? We [do that in the SDK](https://github.com/slackapi/deno-slack-sdk/blob/main/.github/workflows/deno.yml#L22-L23). This way you keep the same testing execution in CI as you do when running locally, and only need to update one place if the test commands change.